starlette/responses.py

```diff
@@ -117,7 +118,8 @@ def set_cookie(
         if max_age is not None:
             cookie[key]["max-age"] = max_age
         if expires is not None:
-            cookie[key]["expires"] = expires
+            expires = typing.cast(typing.Union[datetime, str], expires)
+            cookie[key]["expires"] = self._format_expires(expires)
```

Comment:
Can we simplify this a bit?
```suggestion
            if isinstance(expires, datetime):
                cookie[key]["expires"] = expires.strftime(COOKIE_DATETIME_FORMAT)
            else:
                cookie[key]["expires"] = expires
```
Also, when creating the `COOKIE_DATETIME_FORMAT` constant, would you mind adding a comment above it with the link on which specifies the format? 🙏

Reply:
Done